pipelines/jobs/configurations/jdk11_pipeline_config.groovy

```diff
@@ -166,9 +166,15 @@ class Config11 {
                 additionalFileNameTag: "linuxXL",
                 configureArgs        : '--with-noncompressedrefs --disable-ccache --enable-dtrace=auto'
         ],
+        riscv64Linux      :  [
```

Comment:
We should have a field in here specifying whether we want to run tests or not. I.e. https://github.com/AdoptOpenJDK/openjdk-build/blob/master/pipelines/jobs/configurations/jdk11_pipeline_config.groovy#L7 OR https://github.com/AdoptOpenJDK/openjdk-build/blob/master/pipelines/jobs/configurations/jdk11_pipeline_config.groovy#L99 

If we ARE going to run tests against this, we should have the relevant test jobs created in Jenkins, otherwise there will be nowhere for the tests to run

Reply:
At the moment we do not have a system functional enough to run tests properly, so tests should be disabled - we can add that later